integration-tests/kubernetes-client/src/test/java/io/quarkus/it/kubernetes/client/CustomKubernetesMockServerTestResource.java

```diff
@@ -22,6 +26,24 @@ public void configureMockServer(KubernetesMockServer mockServer) {
                 .andReturn(200, configMapBuilder("cmap2")
                         .addToData("application.yaml", "some:\n  prop4: val4").build())
                 .once();
+
+        mockServer.expect().get().withPath("/api/v1/namespaces/demo/configmaps/cmap3")
+                .andReturn(200, configMapBuilder("cmap3")
+                        .addToData("dummy", "dummyFromDemo")
+                        .addToData("some.prop1", "val1FromDemo")
+                        .addToData("some.prop2", "val2FromDemo")
+                        .addToData("some.prop5", "val5FromDemo")
+                        .addToData("application.properties", "some.prop3=val3FromDemo")
+                        .addToData("application.yaml", "some:\n  prop4: val4FromDemo").build())
+                .once();
+    }
+
+    public static void assertProperty(String propertyName, String expectedValue) {
```

Comment:
I don't think the assertions belong here

Reply:
yes, I moved to use it from other place and finally forgot undo the change.